app/src/main/java/com/ucmobiledevelopment/freeways/service/IncidentService.kt

```diff
@@ -30,21 +37,22 @@ class IncidentService : IIncidentService {
             val result = incidentListDTO.await()?.awaitResponse()?.body()
 
             if (result != null) {
-                if(result.Results[0].isNotEmpty()){
-                    result.Results[0].forEach{
-                        val newIncident: Incident = Incident()
-
-                        newIncident.cityName = it.CITYNAME ?: ""
-                        newIncident.countyId = if(it.COUNTY != null) it.COUNTY.toInt() else 0
-                        newIncident.countyName = it.COUNTYNAME ?: ""
-                        newIncident.stateId = if(it.STATE != null) it.STATE.toInt() else 0
-                        newIncident.stateName = it.STATENAME ?: ""
```

Comment:
Formated the flow control statements (Kotlin convention), corrected the typos. and broke a long line into 2 parts. 
https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements
https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace
https://kotlinlang.org/docs/coding-conventions.html#properties